### DISCUSSION TITLE
Using the `css` helper with the JSX precompile feature for Deno

### DISCUSSION THREAD

**Discussion by stewarthsoj** [honojs/hono#2281] [Q&A] [answered]
Currently, from what I can tell, it's not possible to use the `css` helper with the JSX precompile feature in Deno. The promises are not resolved. Is there a way to make it work?

**Comment by yusukebe** [comment]
Hi @JoshStwrt 

cc: @usualoma Can you answer this question?

**Reply by stewarthsoj** [reply]
Hi @yusukebe, many thanks for taking a look @usualoma

**Comment by usualoma** [comment]
Hi @JoshStwrt
Thanks for your comment!
This is a bug and will be fixed at https://github.com/honojs/hono/pull/2283. Please check back after the release.

**Reply by stewarthsoj** [reply]
Sure, appreciate the reply @usualoma

**Reply by yusukebe** [reply, marked as answer]
@JoshStwrt  I've just released v4.0.8 including the fix by @usualoma . Try it!

**Reply by stewarthsoj** [reply]
@yusukebe @usualoma works well!
